Extensions/Taxonomies/ITaxonomySetupHandler.cs

```diff
@@ -21,5 +21,13 @@ public interface ITaxonomySetupHandler : IEventHandler
         /// <param name="taxonomyName">Names of the Taxonomies to create.</param>
         /// <param name="termNames">Names of the Terms to create and add to the Taxonomy.</param>
         void ApplyTaxonomyCreationWithTerms(string taxonomyName, List<string> termNames);
+
+        /// <summary>
+        /// Applies the previously invoked creation of Taxonomy with Terms and Taxonomy settings.
+        /// </summary>
+        /// <param name="taxonomyName">Names of the Taxonomies to create.</param>

```

Comment:
Csak egy TaxonomyName van, tehát singularnak kéne lennie (ebből van több helyen, function neveket is csekkold ennek mentén).